Library/Formula/td.rb

```diff
@@ -0,0 +1,49 @@
+require "language/go"
+
+class Td < Formula
+  homepage "https://github.com/Swatto/td"
+  revision 1
```

Comment:
Oops, Sorry. Tim's comments didn't exist before I commented, they appeared once I pressed the refresh key. Sorry Tim :sweat_smile: